cluster/migrator/partitionmigration/client/grpc_client.go

```diff
@@ -0,0 +1,68 @@
+// GRPC client for partition migration
+package client
+
+import (
+	"context"
+	"net"
+	"time"
+
+	"google.golang.org/grpc"
+	grpcbackoff "google.golang.org/grpc/backoff"
+	"google.golang.org/grpc/credentials/insecure"
+	"google.golang.org/grpc/keepalive"
+
+	"github.com/rudderlabs/rudder-go-kit/bytesize"
+	"github.com/rudderlabs/rudder-go-kit/config"
+	proto "github.com/rudderlabs/rudder-server/proto/cluster"
+)
+
+// NewPartitionMigrationClient creates a new client for the PartitionMigration gRPC service
+func NewPartitionMigrationClient(target string, conf *config.Config) (PartitionMigrationClient, error) {
+	conn, err := grpc.NewClient(target,
+		grpc.WithTransportCredentials(insecure.NewCredentials()),
```

Comment:
Any security concerns we need to double check with Security team? 
May be we should at least have some authentication mechanism?